test/testAppinfoJsonFileType.js

```diff
@@ -64,5 +64,33 @@ module.exports.appinfojsonfiletype = {
         test.ok(ajft);
         test.ok(!ajft.handles("lappinfo.json"));
         test.done();
+    },
+    testAppinfoJsonFileTypeHandlesJonFalselocaleDir: function(test) {
```

Comment:
Who is Jon and why does this plugin handle him? ;-)

Reply:
Oops. Who is Jon? haha. I fixed them all.